test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/TestResources.cs

```diff
@@ -12,5 +13,10 @@ public static class TestResources
         public static string DisplayAttribute_Description { get; } = Resources.DisplayAttribute_Description;
 
         public static string DisplayAttribute_Name { get; } = Resources.DisplayAttribute_Name;
+
+        public static string DisplayAttribute_Name2 =>
```

Comment:
Give this and `DisplayAttribute_Description2` more meaningful names. Suggest `DisplayAttribute_NamePlusCulture` for this one. Or maybe `DisplayAttribute_CultureSensitiveName`.